lib/perlcritic/Perl/Critic/Policy/OpenQA/SpaceAfterSubroutineName.pm

```diff
@@ -77,7 +79,8 @@ sub _is_block ($token) {
 }
 
 sub _is_only_one_space ($token) {
-    return $token->isa('PPI::Token::Whitespace') && $token->content eq ' ';
+    # We also allow line breaks, e.g. perltidy forces to break up long subroutine headers
+    return $token->isa('PPI::Token::Whitespace') && $token->content =~ m/^[ \n]$/;
```

Comment:
Do we need a better function name? Maybe `is_whitespace_or_newline`? but not sure

Reply:
maybe...